### DISCUSSION TITLE
cannot console.log inside the credentials provider

### DISCUSSION THREAD

**Discussion by firawz** [nextauthjs/next-auth#5370] [Help]
### Question 💬

Hey guys i want to use nextauth credentials providers and there are several problems that i encountered
1. the console.log response from axios did not come out in the terminal when i hit the sign in button..
2. for some reason everytime i reload the pages i get this error 

here is my code within in the [...nextauth].ts

```js
import NextAuth from 'next-auth';
import CredentialsProvider from 'next-auth/providers/credentials';

export default NextAuth({
  session: {
    strategy: 'jwt',
  },
  providers: [
    CredentialsProvider({
      name: 'email&Pass',
      type: 'credentials',
      credentials: {},
      async authorize(credentials, req) {
         console.log('test')
        console.log(credentials);
        const res = await axios.post('login', credentials); // to backend services which work in postman
        console.log(res);

        if (res) {
          return res;
        }
        throw new Error('NOT FOUND');
      },
    }),
  ],
});
```

and this is the code from login in form that i created 

```ts
const clickLogin = async (e: SyntheticEvent) => {
    e.preventDefault();
    console.log({ ...value });
    signIn('credentials', { redirect: false, ...value }); // ...value => {email: 'abc@mail.com', password: '1234'} 
  };
```

for some reason, the console.log in the browser only show `console.log({ ...value });` this one. and there is no console log in the terminal, not even the `console.log('test')`

i was expecting my data of the login request to be console.log and store the data to cookies or session in order to make it work in nextjs getserversideprops for protecting routes

is there anything that i missed? i follow the guide, retry about 4 6th times with next-js example. did not work at all. 
please help me :((

### How to reproduce ☕️

clone this https://github.com/vercel/next.js/tree/canary/examples/with-next-auth repository

run through the tutorial that the documentation provided.
create a simple sign in method and backend services. 
make sure the api work on postman. try to sign in with my logics above 

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by ThangHuuVu** [comment]
Hi there, this sounds like a configuration error to me. Could you check in the Devtools Network tab to see if the login request is there?
It might be because `NEXTAUTH_URL` is missing.

**Reply by firawz** [reply]
my NEXTAUTH_URL=localhost:3000 and for some reason, sometimes it cames out, the client fetch error, sometimes it doesnt.. really doesnt understand why.. 

then when i hit the login button it redirects me to http://localhost:3000/api/auth/error/ even though i have the option {redirect: false} within the sign in function, then i try to slow 3g to capture this network
![image](https://user-images.githubusercontent.com/57997994/190881270-f2f20e05-60d3-4da4-a3dc-f864675c60e1.png)

@ThangHuuVu
